tools/e2e/examples/docker-compose/docker-compose.yaml

```diff
@@ -214,9 +235,222 @@ services:
+  #
+  kuma-example-web-namespace:
+    image: busybox:1.31.0
+    command: ["sleep", "infinity"]
+    expose:
+    - "6060"
+    # The following `ports` configuration was originally a part of `kuma-example-web` service.
+    # However, `ports` settings are incompatible with `network_mode: service:...`
+    # and had to be moved here.
+    ports:
+    - "6060:6060"
+    networks:
+      kuma-example:
+        aliases:
+        - kuma-example-web
+    depends_on:
+    - kuma-control-plane
+    restart: on-failure
+
+  #
+  # Example Web deployed into Kuma mesh.
+  #
+  kuma-example-web:
+    image: kong-docker-kuma-ci-docker.bintray.io/curl
+    command:
+    - nc
+    - -lk
+    - -s
+    - 127.0.0.1
+    - -p
+    - "6060"
+    - -e
+    - curl
+    - --silent
+    - --show-error
+    - --include
+    - --fail
+    - http://localhost:5000/version
+    # the following setting instructs Docker Compose to add `kuma-example-web` container
+    # to the network namespace of `kuma-example-web-namespace` container
+    network_mode: service:kuma-example-web-namespace
+    depends_on:
+    - kuma-example-web-namespace
+    restart: on-failure
+
+  #
+  # Kuma sidecar for the Example Web.
+  #
+  kuma-example-web-sidecar:
+    # image name must be provided via environment variable KUMA_DP_DOCKER_IMAGE
+    image: ${KUMA_DP_DOCKER_IMAGE:-kuma/kuma-dp:latest}
+    command:
+    - run
+    - --log-level=info
+    volumes:
+    - token-example-web:/var/kuma.io/kuma-dp/kuma-example-web
+    environment:
+    - KUMA_CONTROL_PLANE_API_SERVER_URL=http://kuma-control-plane:5681
+    - KUMA_DATAPLANE_MESH=default
+    - KUMA_DATAPLANE_NAME=kuma-example-web
+    - KUMA_DATAPLANE_ADMIN_PORT=9901
+    - KUMA_DATAPLANE_RUNTIME_TOKEN_PATH=/var/kuma.io/kuma-dp/kuma-example-web/token
+    # the following setting instructs Docker Compose to add `kuma-example-web-sidecar` container
+    # to the network namespace of `kuma-example-web-namespace` container
+    network_mode: service:kuma-example-web
+    depends_on:
+    - kuma-example-web
+    restart: on-failure
+
+  #
+  # Auxiliary service (container) for sharing Linux network namespace
+  # between Example Backend v1 and Kuma Sidecar.
+  #
+  kuma-example-backend-v1-namespace:
+    image: busybox:1.31.0
+    command: ["sleep", "infinity"]
+    expose:
+    - "7070"
+    # The following `ports` configuration was originally a part of `kuma-example-backend-v1` service.
+    # However, `ports` settings are incompatible with `network_mode: service:...`
+    # and had to be moved here.
+    ports:
+    - "17070:7070"
+    networks:
+      kuma-example:
+        aliases:
+        - kuma-example-backend-v1
+    depends_on:
+    - kuma-control-plane
+    restart: on-failure
+
+  #
+  # Example Backend v1 deployed into Kuma mesh.
+  #
+  # Notice that we're using Envoy as a simple HTTP server with predefined responses.
+  #
+  kuma-example-backend-v1:
+    image: envoyproxy/envoy-alpine:v1.12.1
```

Comment:
```suggestion
    image: envoyproxy/envoy-alpine:${ENVOY_VERSION:-v1.12.1}
```

Reply:
`${ENVOY_VERSION:-v1.12.1}` would be useful if we were testing different version of `envoy`, but it's not the case here (`envoy` is used as a sample HTTP server).

Trying to minimize amount of YAML developers will have to read, understand and maintain